pysph/tools/interpolator.py

```diff
@@ -1,6 +1,6 @@
 # Standard library imports
 from functools import reduce
-
+interpolator_methods = ['shepard', 'sph', 'order1', 'splash', 'splash_norm']
```

Comment:
This could be a class attribute on the `Interpolator` class, say `Interpolator.METHODS` or something like that?

Reply:
Yes!